tabs/firmware_flasher.js

```diff
@@ -292,7 +293,8 @@ TABS.firmware_flasher.initialize = function (callback) {
 
                         $('a.flash_firmware').removeClass('disabled');
 
-                        if (summary.commit) {
+                        if (summary.commit) {   //woga65:
+                            //$.get('https://api.github.com/repos/Woga65/inav/commits/' + summary.commit, function (data) {                                
```

Comment:
Likewise, I assume this is for testing. The official iNavFlight repos should be used in release trim.

Reply:
Yes, for testing. I will clean this up as soon as all probebly needed changes are discussed and applied.